yaml/degoogle.yml

```diff
@@ -2031,9 +2031,27 @@ mobile applications:
       fdroid: org.blokada.alarm
       eyes: null
       text: >
-        From the website: "If you want to efficiently block ads, trackers, malware, save on your data
+        From the website: If you want to efficiently block ads, trackers, malware, save on your data
```

Comment:
This is a mistake to remove the quote, the closing quote is two lines down.

Reply:
Yikes. My bad. Thanks for spotting it.

Fixed it in the newer commit.